app/event_images/event_images.py

```diff
@@ -0,0 +1,216 @@
+from functools import lru_cache
```

Comment:
Can you please move this file to app/routers/event_images.py?

The images should be in app/static/event_images/...

Reply:
done :) these are the 2 comments you mentioned?